pkg/runtime/runtime.go

```diff
@@ -554,7 +555,9 @@ func (a *DaprRuntime) beginPubSub(name string, ps pubsub.PubSub) error {
 
 			var cloudEvent map[string]interface{}
 			data := msg.Data
-			if rawPayload {
+			if rawPayload && defaultDataAsPayload {
+				cloudEvent = pubsub.FromRawPayload(nil, msg.Topic, name)
```

Comment:
what is the idea with this?

Reply:
I want to avoid base64 encoding when `rawPayload` and `DataAsPayload` are both true. But haven't found a proper way, I'll reconsider this logic later.(`DataAsPayload` is identified by applying matcher to cloudevent)

Reply:
Deleted it for a better review.